### DISCUSSION TITLE
Stone pickers

### DISCUSSION THREAD

**Discussion by Misamangame** [Courseplay/Courseplay_FS22#2169] [Q&A] [answered]
Hello,
Can stone pickers (especially ELHO two others are shit anyway) work as a combine and when they are full and suitable trailer is near by they will go to unload into him?

**Comment by Tensuko** [comment, marked as answer]
Right now not, but they can be emptyed with AutoDrive like a loading wagon.

**Reply by Misamangame** [reply]
Yes I know using them like that, but ofcourse as they have pretty small tank it should be nice if they will be able to empty to wagon next to field as travelling distances adding lots of time to do single field.

**Reply by Tensuko** [reply]
They do not have that kind of overloading function.
What can be done is drive behind them to unload them with a trailer, but not automatically.
That woul requiere a special driver.

**Reply by Misamangame** [reply]
Ok thats unlucky :(.  Yes I was also thinking that other way around when unloader will have to come to unload will be more complicated. No worries then, thx for clarifing that.
